tokio/src/runtime/blocking/mod.rs

```diff
@@ -288,7 +288,7 @@ where
 ///
 /// ```
 /// # async fn docs() {
-/// tokio::executor::blocking::run(move || {
+/// tokio::runtime::blocking::run(move || {
```

Comment:
Similarly, what do you think about `tokio::blocking::run`?

Reply:
I was thinking `tokio::spawn_blocking`. wdyt?

Reply:
That sounds good to me — maybe we rename the in place variant to just `blocking`, so we have `tokio::spawn_blocking` and `tokio::blocking`? although, it's a little weird how one is a verb and the other is an adjective...i'd say `tokio::spawn_blocking` and `tokio::block`, but `block` is too similar to `block_on`...

Reply:
`tokio::spawn_blocking` and `tokio::block_in_place`?

Reply:
Well, it is expected for end users to call `tokio::spawn_blocking`, but `block_in_place` is an "advanced" API that most users should not call. Because of this, my inclination is to keep it out of the root.